go.mod

```diff
@@ -22,8 +22,8 @@ go 1.15
 
 require (
 	github.com/GoogleCloudPlatform/spark-on-k8s-operator v0.0.0-20201215015655-2e8b733f5ad0
-	github.com/apache/incubator-yunikorn-core v0.0.0-20211207190138-7ccbebf8230d
-	github.com/apache/incubator-yunikorn-scheduler-interface v0.11.1-0.20211103035638-29e980cee5f1
+	github.com/apache/incubator-yunikorn-core v0.0.0-20211213184834-b34f64e9fd32
+	github.com/apache/incubator-yunikorn-scheduler-interface v0.12.0
```

Comment:
I'm going to re-tag the latest commit in SI as 0.12.1 for consistency, but it's going to be identical to the previous 0.12.0

Reply:
Ok, will update PR once we have that. Thanks!

Reply:
@craigcondit The 0.12.1 tags for both the core and SI have been created. Feel free to use those here